### PR TITLE
Add a missing string (ca, nb, pl, pt-BR) and add & update Korean translations

### PR DESCRIPTION
**Pull request by te6-in**
This updates some translations and adds new ones for the font options.
I love how strings for new options are being added day by day. 😄

### CODE REVIEW

**Comment by Dimillian** [comment]
Do you have the file on a watcher? Haha. Thanks!

**Comment by te6-in** [comment]
> Do you have the file on a watcher? Haha. Thanks!

I didn't know I could do that! Now I set up one :)

**Comment by te6-in** [comment]
@Dimillian And I just noticed that the `status.action.translate-from-%@` string isn't pre-added from some localization files(`ko`, `pl`, and `ca`). I think people who contribute to translations should keep that in mind or they might miss it out like me.

**Comment by Dimillian** [comment]
I think it's was before those new lang PR was merged. Can you add it in this PR?

**Comment by te6-in** [comment]
Yeah sure. Please take a look

**Comment by Dimillian** [comment]
Thanks a lot!

File: IceCubesApp/Resources/Localization/pt-BR.lproj/Localizable.strings [hunk 1]
```diff
@@ -291,6 +291,7 @@
 
 // MARK: Package: Status
 "status.action.translate" = "Traduzir";
+"status.action.translate-from-%@" = "Translate from %@";
```
Comment (martimlobao) [inline IceCubesApp/Resources/Localization/pt-BR.lproj/Localizable.strings:294]: shouldn't this be in portuguese?
```suggestion
"status.action.translate-from-%@" = "Traduzir de %@";
```

File: IceCubesApp/Resources/Localization/pl.lproj/Localizable.strings [hunk 2]
```diff
@@ -291,6 +291,7 @@
 
 // MARK: Package: Status
 "status.action.translate" = "Przetłumacz";
+"status.action.translate-from-%@" = "Translate from %@";
```
Comment (martimlobao) [inline IceCubesApp/Resources/Localization/pl.lproj/Localizable.strings:294]: shouldn't this be in polish?

File: IceCubesApp/Resources/Localization/nb.lproj/Localizable.strings [hunk 3]
```diff
@@ -291,6 +291,7 @@
 
 // MARK: Package: Status
 "status.action.translate" = "Oversett";
+"status.action.translate-from-%@" = "Translate from %@";
```
Comment (martimlobao) [inline IceCubesApp/Resources/Localization/nb.lproj/Localizable.strings:294]: norwegian?

**Review by martimlobao — COMMENTED** [review]
i think some of the translations here are wrong, or am i missing something?

File: IceCubesApp/Resources/Localization/nb.lproj/Localizable.strings (hunk 3 above)
Comment (espenbye) [reply IceCubesApp/Resources/Localization/nb.lproj/Localizable.strings:294]: I have translated the new strings (including this) for Norwegian in PR #537

File: IceCubesApp/Resources/Localization/pt-BR.lproj/Localizable.strings (hunk 1 above)
Comment (te6-in) [reply IceCubesApp/Resources/Localization/pt-BR.lproj/Localizable.strings:294]: Yeah, what I added (except for the Korean file) are just English placeholders.
I believe the translation works have been done this way so far (English placeholder added → translated by the speakers).

Will it be okay if I commit your suggestion for this Portuguese one? Please let me know.

File: IceCubesApp/Resources/Localization/ca.lproj/Localizable.strings [hunk 4]
```diff
@@ -290,6 +290,7 @@
 
 // MARK: Package: Status
 "status.action.translate" = "Tradueix";
+"status.action.translate-from-%@" = "Translate from %@";
```
[the remark this answers is not in the document]
Comment (Dimillian) [reply IceCubesApp/Resources/Localization/ca.lproj/Localizable.strings:293]: It's ok, it's just placeholder for missing strings.

File: IceCubesApp/Resources/Localization/pt-BR.lproj/Localizable.strings (hunk 1 above)
Comment (martimlobao) [reply IceCubesApp/Resources/Localization/pt-BR.lproj/Localizable.strings:294]: Sorry, missed this and just saw you already merged, but yeah, perfectly fine to commit this suggestion 🙂

**Pull request merged**